securedrop/db.py

```diff
@@ -251,10 +251,17 @@ class Journalist(Base):
         "JournalistLoginAttempt",
         backref="journalist")
 
+    MIN_USERNAME_LEN = 8
+
     def __init__(self, username, password, is_admin=False, otp_secret=None):
+        if len(username) < self.MIN_USERNAME_LEN:
+            raise InvalidUsernameException(
+                    'Username "{}" is too short'.format(username))
```

Comment:
it would help l10n to have 'Username "{username}" must be at least {min} character long".format(username=username, min=self.MIN_USERNAME_LEN)